src/clr-angular/forms/combobox/combobox-container.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, OnDestroy, Optional, ContentChild, ElementRef, ViewChild } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { NgControl } from '@angular/forms';
+
+import { IfErrorService } from '../common/if-error/if-error.service';
+import { NgControlService } from '../common/providers/ng-control.service';
+import { ControlIdService } from '../common/providers/control-id.service';
+import { ControlClassService } from '../common/providers/control-class.service';
+import { DynamicWrapper } from '../../utils/host-wrapping/dynamic-wrapper';
+import { LayoutService } from '../common/providers/layout.service';
+import { ClrLabel } from '../common/label';
+import { ComboboxOffsetService } from './providers/combobox-offset.service';
+
+@Component({
+  selector: 'clr-combobox-container',
+  template: `
+    <ng-content select="label"></ng-content>
+    <label *ngIf="!label && addGrid()"></label>
+    <div class="clr-control-container" [ngClass]="controlClass()" #controlContainer>
+      <ng-content select="clr-combobox"></ng-content>
+      <clr-icon *ngIf="invalid" class="clr-validate-icon" shape="exclamation-circle" aria-hidden="true"></clr-icon>
+      <ng-content select="clr-control-helper" *ngIf="!invalid"></ng-content>
+      <ng-content select="clr-control-error" *ngIf="invalid"></ng-content>
+    </div>
+  `,
+  // needed for the proper calculation of popover offset that we do to keep label always visible
+  styles: [':host { align-items: stretch; }'],
```

Comment:
I _think_ this should be in the scss file and targeted to the `clr-combobox-container` element.

Reply:
Done!